core-services/prow/02_config/_config.yaml

```diff
@@ -665,6 +665,7 @@ tide:
     - openshift/cluster-svcat-controller-manager-operator
     - openshift/cluster-update-keys
     - openshift/cluster-version-operator
+    - openshift/compliance-operator
```

Comment:
You should also add the repo to the second query, the one related to release branches (around L462)